packages/integration-sdk-runtime/src/execution/uploader.ts

```diff
@@ -133,17 +133,13 @@ export interface CreatePersisterApiStepGraphObjectDataUploaderParams {
   stepId: string;
   synchronizationJobContext: SynchronizationJobContext;
   uploadConcurrency: number;
-  uploadBatchSize?: number;
-  uploadRelationshipsBatchSize?: number;
   uploadBatchSizeInBytes?: number;
 }
 
 export function createPersisterApiStepGraphObjectDataUploader({
   stepId,
   synchronizationJobContext,
   uploadConcurrency,
-  uploadBatchSize,
```

Comment:
I know this two are used in https://github.com/JupiterOne/jupiter-managed-integration/blob/1391f095c258cfe7f55e68d459c025623eba3351/packages/runtime/src/util/uploader.ts#L28 but shouldn't break anything, maybe we also want to cleanup there